BootloaderCommonPkg/Library/UsbDeviceLib/UsbDeviceLib.c

```diff
@@ -42,13 +42,13 @@ UsbDeviceDriverSupported (
 
   UsbInfo = &mUsbDeviceInfo;
 
-  Address = MmPciAddress ( 0, UsbInfo->XdciDeviceAddress.Bus, UsbInfo->XdciDeviceAddress.Device,
+  Address = MM_PCI_ADDRESS (UsbInfo->XdciDeviceAddress.Bus, UsbInfo->XdciDeviceAddress.Device,
```

Comment:
why not just  define MmPciAddress as macro.  In this way it needs much less changes.  Also it is easy to compare with original EDKII code.

Reply:
By the coding style of EDK2: Use all capital letters for both #define and typedef declarations.
This clearly differentiates static declarations from dynamic data types. So just not use MmPciAddress().

Reply:
Understand the coding style requirement.  But these changes make it difficult to compare with the origin files from EDK2.  If it is a concern,  we can add this to the common lib as a real function.

Reply:
Roll back changes to common USB lib. only update platform code for MmPciAddress() in this patch.